src/helpers/author.ts

```diff
@@ -0,0 +1,23 @@
+import { Author } from 'lib/post';
+
+const getAvatarUrl = (author: Author, size = 128): string => {
+  if (author.username) {
+    return `https://github.com/${author.username}.png?size=${size}`;
+  }
+
+  return author.avatarUrl || '';
+};
+
+const getAuthorName = (author: Author): string => {
+  if (author.name) {
+    return author.name;
+  }
+
+  if (author.username) {
+    return author.username;
+  }
+
+  return '';
```

Comment:
@hoangmirs Should we make it like this?
Because I see you use a shorter like [here](https://github.com/nimblehq/til-web/pull/38/files#diff-e7a5cbf09399192d765534e6b89391dc64f0fd932e9e0a71404dd70190bd788dR8)
and it returns as it has 🤔

```suggestion
  return author.name || author.username || '';
```

Reply:
Yeah, changed in https://github.com/nimblehq/til-web/pull/38/commits/62f21752e9e19cae6fcca925a45aa2315819ab18